independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/ClientProxyGenerator.java

```diff
@@ -98,7 +98,14 @@ public ClientProxyGenerator(Predicate<DotName> applicationClassPredicate) {
 
             // Exceptions
             for (Type exception : method.exceptions()) {
-                forward.addException(exception.asClassType().toString());
+                Type.Kind kind = exception.kind();
+                if (Type.Kind.CLASS.equals(kind)) {
```

Comment:
Hm, so I'm wondering:
1. Do we actually need to check the specific type or just always use `toString()`?
2. Should we add some tests? ;-)

Reply:
1. that's a good point, we can just use `toString()`
2. I thought this change a bit too trivial to write a test for it, feel free to add one ;)